help/keyboard-shortcuts.md

```diff
@@ -131,7 +131,7 @@ in the Zulip app to add more to your repertoire as needed.
   selected message (outlined in blue). Same behavior as clicking on the
   message.
 
-* **Quote and reply to message**: <kbd>></kbd>
+* **Quote message**: <kbd>></kbd>
 
```

Comment:
The forward shortcut should be added here, as well as in the `?` menu.

Reply:
Oops, i missed that one, I've added it there now. The `?` menu is already done I think, I assume you mean this one:
![image](https://github.com/user-attachments/assets/7be0e8f7-311e-4e0f-9dae-12b040316e50)
![image](https://github.com/user-attachments/assets/10440f9f-63e2-4e15-aeb3-61c6b6c9094b)